calendar.yaml

```diff
@@ -259,3 +259,18 @@
       San Diego Convention Center – Sails Pavilion – O'Reilly Booth–S25
   organizer: jpatanooga
   
+
+- id: kf024
+  name: Model Management SIG Call
+  date: 26/5/2020
+  time: 8:00am-9:00am
+  frequency: bi-weekly
+  video: https://meet.google.com/dxz-zthy-xjn
+  description:
+    - |
+      Notes & agenda: https://docs.google.com/document/d/1R2r1z5O4USpn3BW29-4blCoPueB3p2LAjvdKa97fT7U/edit?usp=sharing
+
+      Join at: https://zoom.us/j/799749911
```

Comment:
I don't think you will have access to this zoom call. Do you have a different way to host the meeting e.g. Google meets where you can host the meeting?

Reply:
Sorry that was a copy paste error. The correct link is under "video" above